### ISSUE TITLE
Alignment build error on Debian

### ISSUE THREAD

**Issue by yuppox** [minoca/os#157]
Using Debian 10.3 x86_64
I'm using the latest version from github and the Linux (64-bit) toolchain (as provided from minocacorp.com).

```
os/include/minoca/kernel/x86.h:418:1: error: alignment 1 of ‘struct _SIGNAL_CONTEXT_X86’ is less than 64 [-Werror=packed-not-a]
 } PACKED SIGNAL_CONTEXT_X86, *PSIGNAL_CONTEXT_X86;
 ^
os/include/minoca/kernel/x86.h:417:17: error: ‘FpuContext’ offset 116 in ‘struct _SIGNAL_CONTEXT_X86’ isn’t aligned to 64 [-We]
     FPU_CONTEXT FpuContext;
```

**Comment by evangreen** [comment]
Thanks for reporting this. It looks like GCC added some new warnings. I added some fixes, can you sync to 460ae51663e02f607f0738521cc2488e0866e70c and see how that works for you?

**Comment by evangreen** [comment]
Closing this as Fixed, feel free to reopen if this is still an issue.
